src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -760,91 +952,202 @@ const Packages = () => {
     } else {
       dispatch(addRecommendedRepository(epelRepo.data[0]));
     }
-    dispatch(addPackage(isSelectingPackage!));
+    if (isSelectingPackage) {
+      dispatch(addPackage(isSelectingPackage!));
+    }
+    if (isSelectingGroup) {
+      dispatch(addGroup(isSelectingGroup!));
+    }
     setIsRepoModalOpen(!isRepoModalOpen);
   };
 
   const composePkgTable = () => {
-    return transformedPackages
-      .slice(computeStart(), computeEnd())
-      .map((pkg, rowIndex) => (
-        <Tr key={`${pkg.name}-${rowIndex}`} data-testid="package-row">
-          <Td
-            select={{
-              isSelected: packages.some((p) => p.name === pkg.name),
-              rowIndex: rowIndex,
-              onSelect: (event, isSelecting) =>
-                handleSelect(pkg, rowIndex, isSelecting),
-            }}
-          />
-          <Td>{pkg.name}</Td>
-          <Td>
-            {pkg.summary ? (
-              pkg.summary
+    let rows: ReactElement[] = [];
+    rows = rows.concat(
+      transformedGroups
+        .slice(computeStart(), computeEnd())
+        .map((grp, rowIndex) => (
+          <Tr key={`${grp.name}-${rowIndex}`} data-testid="package-row">
+            <Td
+              select={{
+                isSelected: groups.some((g) => g.name === grp.name),
+                rowIndex: rowIndex,
+                onSelect: (event, isSelecting) =>
+                  handleGroupSelect(grp, rowIndex, isSelecting),
+              }}
+            />
+            <Td>
+              @{grp.name}
+              <Popover
+                bodyContent={
+                  <div style={{ height: '40em', overflow: 'scroll' }}>
+                    <Table variant="compact">
+                      <Thead>
+                        <Tr>
+                          <Th>Included packages</Th>
+                        </Tr>
+                      </Thead>
+                      <Tbody>
+                        {grp.package_list.map((pkg) => (
```

Comment:
Also maybe we could make "Included packages" a heading of the popover and make only the table body scrollable, but that's very nit picky and I'm happy to open a follow up with suggestion later.